src/main/java/com/jpa/entity/Developer.java

```diff
@@ -0,0 +1,43 @@
+package com.jpa.entity;
+
+import com.jpa.listeners.DeveloperListener;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@Entity
+@EntityListeners(DeveloperListener.class)
+@Table(name = "developers")
+public class Developer extends AbstractIdentifiableObject{
```

Comment:
https://stackoverflow.com/questions/7506802/using-wrapper-integer-class-or-int-primitive-in-hibernate-mapping